eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServer.java

```diff
@@ -55,98 +64,70 @@ public RegionWSServer() {
 //    @ApiOperation(httpMethod = "GET", value = "Retrieves all the variants from region", response = QueryResponse.class)
     public QueryResponse getVariantsByRegion(@PathVariable("regionId") String regionId,
                                              @RequestParam(name = "species") String species,
-                                             @RequestParam(name = "studies", required = false) String studies,
+                                             @RequestParam(name = "studies", required = false) List<String> studies,
                                              @RequestParam(name = "annot-ct", required = false) List<String> consequenceType,
-                                             @RequestParam(name = "maf", defaultValue = "") String maf,
-                                             @RequestParam(name = "polyphen", defaultValue = "") String polyphenScore,
-                                             @RequestParam(name = "sift", defaultValue = "") String siftScore,
-                                             @RequestParam(name = "ref", defaultValue = "") String reference,
-                                             @RequestParam(name = "alt", defaultValue = "") String alternate,
-                                             @RequestParam(name = "miss_alleles", defaultValue = "") String missingAlleles,
-                                             @RequestParam(name = "miss_gts", defaultValue = "") String missingGenotypes,
-                                             @RequestParam(name = "histogram", defaultValue = "false") boolean histogram,
-                                             @RequestParam(name = "histogram_interval", defaultValue = "-1") int interval,
-                                             @RequestParam(name = "merge", defaultValue = "false") boolean merge,
+                                             @RequestParam(name = "maf", defaultValue = "", required = false) String maf,
+                                             @RequestParam(name = "polyphen", defaultValue = "", required = false) String polyphenScore,
+                                             @RequestParam(name = "sift", defaultValue = "", required = false) String siftScore,
```

Comment:
Call me old fashioned, but I prefer a RAII approach, check only one in initialization, otherwise we tend to fill the code with checkings. Also, another way to do it even better would be add JavaEE standard validation in the parameters and let spring handle the rest... But that may be much work right now and can be done later.